### ISSUE TITLE
Minor: yum install on Centos 8 compains of missing perl module.

### ISSUE THREAD

**Issue by bhyde** [tailscale/tailscale#345]
This does not appear to lead to any further malfunction...

Notice in the longer transcript below this bit:
```
Modular dependency problem:

 Problem: conflicting requests
  - nothing provides module(perl:5.26) needed by module perl-DBI:1.641:8010020191113222731:16b3ab4d-0.x86_64
```
Longer transcript:
```
/ssh:bbctun:/ #$ dnf config-manager --add-repo https://pkgs.tailscale.com/stable/centos/8/tailscale.repo
Adding repo from: https://pkgs.tailscale.com/stable/centos/8/tailscale.repo
/ssh:bbctun:/ #$ ls
bin   dev  home  lib64	     media  opt   root	sbin  sys  usr
boot  etc  lib	 lost+found  mnt    proc  run	srv   tmp  var
/ssh:bbctun:/ #$ mkdir /var/cache/tailscale
/ssh:bbctun:/ #$ ls -l /var/cache/tailscale
total 0
/ssh:bbctun:/ #$ yum install tailscale
CentOS-8 - AppStream                             16 kB/s | 4.3 kB     00:00    
CentOS-8 - Base                                  43 kB/s | 3.9 kB     00:00    
CentOS-8 - Extras                               490  B/s | 1.5 kB     00:03    
CentOS-8 - PowerTools                            44 kB/s | 4.3 kB     00:00    
ELRepo.org Community Enterprise Linux Repositor 187  B/s | 2.9 kB     00:16    
Extra Packages for Enterprise Linux Modular 8 -  50 kB/s |  20 kB     00:00    
Extra Packages for Enterprise Linux 8 - x86_64   30 kB/s |  20 kB     00:00    
Tailscale stable                                791  B/s | 800  B     00:01    
Tailscale stable                                6.7 kB/s | 3.1 kB     00:00    
Importing GPG key 0x957F5868:
 Userid     : "Tailscale Inc. (Package repository signing key) <info@tailscale.com>"
 Fingerprint: 2596 A99E AAB3 3821 893C 0A79 458C A832 957F 5868
 From       : https://pkgs.tailscale.com/stable/centos/8/repo.gpg
Is this ok [y/N]: y
Tailscale stable                                1.2 kB/s | 1.8 kB     00:01    
Dependencies resolved.
================================================================================
 Package          Architecture  Version           Repository               Size
================================================================================
Installing:
 tailscale        x86_64        0.98.0-0          tailscale-stable         12 M

Transaction Summary
================================================================================
Install  1 Package

Total download size: 12 M
Installed size: 22 M
Is this ok [y/N]: y
Downloading Packages:
tailscale_0.98-0_x86_64.rpm                     5.2 MB/s |  12 MB     00:02    
--------------------------------------------------------------------------------
Total                                           5.2 MB/s |  12 MB     00:02     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1 
  Installing       : tailscale-0.98.0-0.x86_64                              1/1 
  Running scriptlet: tailscale-0.98.0-0.x86_64                              1/1 
  Verifying        : tailscale-0.98.0-0.x86_64                              1/1 

Installed:
  tailscale-0.98.0-0.x86_64                                                     

Complete!
/ssh:bbctun:/ #$ yum history list
Modular dependency problem:

 Problem: conflicting requests
  - nothing provides module(perl:5.26) needed by module perl-DBI:1.641:8010020191113222731:16b3ab4d-0.x86_64
ID     | Command line             | Date and time    | Action(s)      | Altered
-------------------------------------------------------------------------------
     8 | install tailscale        | 2020-05-01 15:38 | Install        |    1   
     7 | install -y squid httpd-t | 2020-05-01 14:00 | Install        |   54   
     6 | install -y elrepo-releas | 2020-05-01 13:59 | Install        |    1   
     5 | install -y epel-release  | 2020-05-01 13:59 | Install        |    1   
     4 | install -y psmisc nmap n | 2020-05-01 13:59 | Install        |    1   
     3 | update -y                | 2020-05-01 13:53 | I, U           |   68   
     2 | -y install parted vim cu | 2020-03-09 17:04 | Install        |    3   
     1 |                          | 2020-03-09 16:56 | Install        |  603 EE
/ssh:bbctun:/ #$ yum history info 8
Modular dependency problem:

 Problem: conflicting requests
  - nothing provides module(perl:5.26) needed by module perl-DBI:1.641:8010020191113222731:16b3ab4d-0.x86_64
Transaction ID : 8
Begin time     : Fri 01 May 2020 03:38:49 PM UTC
Begin rpmdb    : 658:c795425f4df4045af70dff7c8100218d33a9f7df
End time       : Fri 01 May 2020 03:38:50 PM UTC (1 seconds)
End rpmdb      : 659:c795425f4df4045af70dff7c8100218d33a9f7df
User           : root <root>
Return-Code    : Success
Releasever     : 8
Command Line   : install tailscale
Packages Altered:
    Install tailscale-0.98.0-0.x86_64 @tailscale-stable
/ssh:bbctun:/ #$ 
```

Misc:
```
grep . /etc/*-release
/etc/centos-release:CentOS Linux release 8.1.1911 (Core) 
/etc/os-release:NAME="CentOS Linux"
/etc/os-release:VERSION="8 (Core)"
/etc/os-release:ID="centos"
/etc/os-release:ID_LIKE="rhel fedora"
/etc/os-release:VERSION_ID="8"
/etc/os-release:PLATFORM_ID="platform:el8"
/etc/os-release:PRETTY_NAME="CentOS Linux 8 (Core)"
/etc/os-release:ANSI_COLOR="0;31"
/etc/os-release:CPE_NAME="cpe:/o:centos:centos:8"
/etc/os-release:HOME_URL="https://www.centos.org/"
/etc/os-release:BUG_REPORT_URL="https://bugs.centos.org/"
/etc/os-release:CENTOS_MANTISBT_PROJECT="CentOS-8"
/etc/os-release:CENTOS_MANTISBT_PROJECT_VERSION="8"
/etc/os-release:REDHAT_SUPPORT_PRODUCT="centos"
/etc/os-release:REDHAT_SUPPORT_PRODUCT_VERSION="8"
/etc/redhat-release:CentOS Linux release 8.1.1911 (Core) 
/etc/system-release:CentOS Linux release 8.1.1911 (Core) 
```
Slightly related to https://github.com/tailscale/tailscale/issues/341#issuecomment-622441238

**Comment by danderson** [comment]
Offhand, not seeing how this is on us, we don't declare any Perl dependencies in the tailscale package. I suppose either iproute2 or iptables could be pulling them in, but that also seems hard to follow...

Putting it on the list of CentOS 8 things to investigate, thanks.

**Comment by bhyde** [comment]
Given that, and that nobody will notice unless they are so fastidious that they check the history.  "Ignore it" say I!

**Comment by bradfitz** [comment]
Sure this error isn't leftover unhappy state from something you installed or uninstalled prior?

**Comment by bhyde** [comment]
It is certainly left over state :).

Turns out that msg appears on stderr, so it's possibly of independent of the `yum history`, emits it.

So when did it first show up? -  In /var/log/dnf.log is the msg's debut is just the Tailscale install has finished.   That was when I did my first history cmd.  But who complained doesn't tell us who gifted us with the unhappy state.

Consulting with the web of hearsay I'm enticed into doing: `dnf module reset perl-*` and that eliminated the unhappy state.

I can see that perl-DBI was installed as a side effect of installing squid, http-tools, and patch; before I installed wireguard (a different project) and then tailscale.

I'd recommend closing this and if I happen to discover something that points a finger at Tailscale more clearly I'll be back.
